test-app/tests/integration/components/headless-form-test.gts

```diff
@@ -247,10 +249,141 @@ module('Integration Component headless-form', function (hooks) {
     });
   });
 
+  module('field.radio', function () {
```

Comment:
radio group next? :D

Reply:
yeah, well... I have a checkbox group on my (mental) backlog, but what would a radio group be?

Reply:
I guess the default way a radio input should be used? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

But! I just found a the test that covers this case: https://github.com/CrowdStrike/ember-headless-form/pull/16/files/dd2315f954edc127e74df0afb01b3ec6b033461f#diff-81f4c9b55c57a592e716604c633b7f95d5a5554742954960cb7bd63be78980ecR398 

So nothing to do! (other than docs, later, I suppose (I'm working on a generator for that part 🙃 ))

Reply:
Yeah, that's what I meant basically, radios only work as a group, a single radio makes no sense. For checkboxes that's different, we already have a single checkbox (representing a boolean), but a group of checkboxes (representing an array of string values - or a `<Record<string, boolean>`? 🤔 ) is also a use case we should support.